radixdlt-java/src/main/java/org/radix/serialization2/mapper/JacksonCborMapper.java

```diff
@@ -76,6 +78,11 @@ public static JacksonCborMapper create(SerializerIds idLookup, FilterProvider fi
 			JacksonCodecConstants.U20_VALUE,
 			UInt256::toByteArray
 		));
+		cborModule.addSerializer(RadixResourceIdentifer.class, new JacksonCborObjectBytesSerializer<>(
+			RadixResourceIdentifer.class,
+			JacksonCodecConstants.RRI_VALUE,
+			id -> id.toString().getBytes(RadixConstants.STANDARD_CHARSET)
```

Comment:
Same comment as on `RadixCore`, would like a `to` and `from` `byte[]` function in `RadixResourceIdentifier`.